config/initializers/secure_headers.rb

```diff
@@ -13,9 +13,14 @@
     :enforce     => true,
     :default_src => 'self',
     :frame_src   => 'self',
-    :connect_src => 'self ws: wss:',
+    :connect_src => %w(self ws: wss:),
     :style_src   => 'inline self',
-    :script_src  => 'eval inline self',
-    :img_src     => ['self', '*.gravatar.com']
+    :script_src  => %w(eval inline self),
+    :img_src     => %w(self *.gravatar.com)
   }
+  if Rails.env.development? #allow webpack dev server provided assets
+    dev_server = "http://#{::Rails.configuration.webpack.dev_server.host}:#{::Rails.configuration.webpack.dev_server.port}"
```

Comment:
note, in recent version of webpack-rails the above dev_server.host is a proc.
i had to change the above line to:
```ruby
dev_server = "http://#{::Rails.configuration.webpack.dev_server.host.call}:#{::Rails.configuration.webpack.dev_server.port}"
```